docs/src/theme/DocVersionBanner/index.js

```
@@ -0,0 +1,123 @@
+      {'This is documentation for {siteTitle} {versionLabel}.'}
+    </Translate>
+  )
+}
+
+const BannerLabelComponents = {
+  unreleased: UnreleasedVersionLabel,
+  unmaintained: UnmaintainedVersionLabel,
+}
+
+function BannerLabel(props) {
+  const BannerLabelComponent = BannerLabelComponents[props.versionMetadata.banner]
```

ESLint

[error] Must use destructuring props assignment

(react/destructuring-assignment)